### PR TITLE
Add japanese translation

### PR DESCRIPTION
**Pull request by bravewood**
Hi!
I'm sending Japanese translation for Symfony Demo application.
If you'd want to any changes to this, please comment me!
Thanks.

### CODE REVIEW

File: app/Resources/translations/messages.ja.xliff [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                <target>ログアウト</target>
+            </trans-unit>
+
+            <trans-unit id="post.to_publish_a_comment">
+                <source>post.to_publish_a_comment</source>
+                <target>コメントを投稿する</target>
+            </trans-unit>
+            <trans-unit id="post.num_comments">
+                <source>post.num_comments</source>
+                <target>%count% コメント|%count% comments</target>
```
Comment (javiereguiluz) [inline app/Resources/translations/messages.ja.xliff]: `%count% コメント|%count% comments` -> `%count% コメント|%count% 注釈` ?

File: app/Resources/translations/messages.ja.xliff [hunk 2]
```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="ja" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="note">
+                <source>note</source>
+                <target>NOTE</target>
```
Comment (bocharsky-bw) [inline app/Resources/translations/messages.ja.xliff]: I think we could translate it, maybe  "ノート"?

File: app/Resources/translations/messages.ja.xliff [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+                <target>内容を編集</target>
+            </trans-unit>
+
+            <trans-unit id="menu.post_list">
+                <source>menu.post_list</source>
+                <target>記事一覧</target>
+            </trans-unit>
+            <trans-unit id="menu.back_to_blog">
+                <source>menu.back_to_blog</source>
+                <target>blogへ戻る</target>
```
Comment (bocharsky-bw) [inline app/Resources/translations/messages.ja.xliff]: Could we translate "blog" as "ブログ"?

**Comment by bravewood** [comment]
@javiereguiluz @bocharsky-bw Thanks for your comments. Sorry, I just fixed some points which were pointed out.

**Comment by bocharsky-bw** [comment]
Thanks, it looks good for me

File: app/Resources/translations/messages.ja.xliff (hunk 1 above)
Comment (bravewood) [inline app/Resources/translations/messages.ja.xliff]: In Japanese, 'コメント' is used for both singular comment and plural comments.
So, I think %count% コメント|%count% comments will be %count% コメント|%count% コメント

File: app/Resources/translations/messages.ja.xliff [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+                <target>ログアウト</target>
+            </trans-unit>
+
+            <trans-unit id="post.to_publish_a_comment">
+                <source>post.to_publish_a_comment</source>
+                <target>コメントを投稿する</target>
+            </trans-unit>
+            <trans-unit id="post.num_comments">
+                <source>post.num_comments</source>
+                <target>%count% コメント|%count% コメント</target>
```
Comment (bocharsky-bw) [inline app/Resources/translations/messages.ja.xliff]: Last my point... I think in `ja` we could escape from nplurals=2 in this case.

Could you change this line to: `<target>%count% コメント</target>` ?

My point based on http://localization-guide.readthedocs.org/en/latest/l10n/pluralforms.html#ja

**Comment by bravewood** [comment]
@bocharsky-bw Thanks for pointing out my mistake. I just changed that point!

**Comment by bocharsky-bw** [comment]
:+1:

**Comment by javiereguiluz** [comment]
@bravewood thank you very much for this translation! *どうもありがとう*

**Pull request closed without merging**